lib/sensors/stripe_sensor.hpp

```diff
@@ -0,0 +1,20 @@
+#include <optional>
+
+#include <io/hardware_gpio.hpp>
+
+#include <core/logger.hpp>
+
+namespace hyped::sensors {
+    class Sensor{
+        public:
+            Sensor(const std::uint8_t newPin);
+            ~Sensor();
+
+            int getStripeCount();
+            void updateStripes();
+        private:
+            std::uint8_t pin;
+            int stripeCount = 0;
+            hyped::core::ILogger &log_;

```

Comment:
Class data members are named using lower snake and a trailing underscore `foo_bar_` so `pin_` and `stripe_count_`

Reply:
as an aside, general variables are named as `foo_bar` and `constexpr`s as `kFooBar`